R/get_ems_data.R

```diff
@@ -101,10 +102,12 @@ get_ems_data <- function(which = "2yr", n = Inf, cols = "wq", force = FALSE, ask
                                  " ems data at", rems_data_dir(), ". Is that okay?"))
     }
     ret <- update_cache(which = which, n = n, cols = cols)
-  } else {
-    message("Fetching data from cache...")
-    ret <- ._remsCache_$get(which)[, cols]
+    if(check_only) return(NULL)
+    return(add_rems_type(ret, which))
   }
+  if(check_only) return(NULL)
```

Comment:
I think we only need one instance of `if(check_only) return(NULL)`, and that would be above line 99. I don't think there's any point in going into `if (update) ...` if it's just going to exit and return `NULL` (or rather `invisible(TRUE)`), see other comment

Reply:
yes makes sense to me